codeDeploy/systemctlRestartService.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+SERVICE=${APPLICATION_NAME}
+
+echo "Restarting ${SERVICE} (${APPLICATION_NAME})"
```

Comment:
Not exactly important but as $SERVICE = $APPLICATION_NAME.. you don't really need the bracketed bit. Happy for you to leave it :-)